types/three/three-core.d.ts

```diff
@@ -5110,6 +5110,7 @@ export class Mesh extends Object3D {
     drawMode: TrianglesDrawModes;
     morphTargetInfluences?: number[];
     morphTargetDictionary?: { [key: string]: number; };
+	isMesh: boolean;
```

Comment:
Was this meant to be part of the PR? It might be better to add it in a separate PR

Reply:
Also note that the indentation is off here.